.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "printWidth": 120,
+    "singleQuote": false,
+    "tabWidth": 4,
+    "endOfLine": "lf"
```

Comment:
Would this be necessary with a `.gitattributes` file with `* text=auto`?

Reply:
I don't know... but IF we change something, we should do it everywhere else... 

e.g., in https://github.com/eclipse-thingweb/node-wot/blob/master/.prettierrc.json also

Reply:
> I don't know... but IF we change something, we should do it everywhere else...

True :) I just remembered that we had some issues with the line endings in the WoT repositories that could be resolved with the `.gitattributes` file. But I guess just using `lf` via prettier should be fine.

I also had to think about whether the config could be put in a single repository to be sharable across projects, but maybe that is a bit overkill, at least at the moment.

Reply:
I can copy over https://github.com/w3c/wot-thing-description/blob/main/.gitattributes for example

Reply:
> I can copy over https://github.com/w3c/wot-thing-description/blob/main/.gitattributes for example

Oh, yeah, that sounds good :)